salalib/ivga.h

```diff
@@ -17,17 +17,17 @@
 
 // Interface to handle different kinds of VGA analysis
 
-#include "salalib/pointdata.h"
+#include "salalib/mgraph.h"
 #include "salalib/options.h"
+#include "salalib/pointdata.h"
 
 #include "genlib/comm.h"
 
 #include <string>
 
-class IVGA
-{
-public:
+class IVGA {
+  public:
     virtual std::string getAnalysisName() const = 0;
```

Comment:
This should probably return a `string const &` rather than a copy.

Reply:
This is used like this:
`std::string getAnalysisName() const override { return "Local Visibility Analysis"; }`
so there's no object to return a reference to..